app/templates/components/public/copyright-item.hbs

```diff
@@ -3,7 +3,9 @@
   <br>
 {{/if}}
 <div class='copyright text'>
-  <p>
-    {{t 'This event is licenced under'}} <a href="{{copyright.licenceUrl}}"> {{copyright.licence}} </a>.
-  </p>
+{{#if copyright.licence}}
+    <p>{{t 'This event is licenced under'}} <a href="{{copyright.licenceUrl}}"> {{copyright.licence}} </a>.</p>
+  {{else}}
```

Comment:
don't need the else clause. Just use the if.

Reply:
@CosmicCoder96 Done with the requested changes